docs/keys-service.md

```diff
@@ -46,11 +48,11 @@
 
 ---
 
-## Get Existing Symmetric Key
+### Get Existing Symmetric Key
 
-`GET /key/{keyId}`
+`GET /key/{keyId}?api-version=2020-09-01`
```

Comment:
2020-09-01 is our current version. How about we specify the format here only?

Reply:
The point of the API version is to have URLs that are not necessarily the same across all API versions. So the URL is not generic on the API version.

Eg if 2020-11-01 removes this URL, then it won't be correct to use `{apiVersion}` for this one.

The downside is that if this URL is *the same* for 2020-11-01, we'd have to duplicate the URL, but that's something we'd have to do anyway because there'd be separate docs for each API version.

Reply:
If we are to have separate docs for each API version, we are good.

Reply:
Yes. I expect we'll remove these docs once we have the swagger specs and generate them from the swagger specs, and the swagger specs have to be one spec per API version like we have in the iotedge repo.